docs/dev.rst

```diff
@@ -180,7 +183,7 @@ To check whether you have used the correct syntax, you should run ``tutor mounts
 
   $ tutor mounts add /path/to/edx-platform
   $ tutor mounts list
-  - name: /home/data/regis/projets/overhang/repos/edx/edx-platform
+  - name: /home/data/regis/projects/overhang/repos/edx/edx-platform
```

Comment:
This is actually my own home path :sweat_smile: please replace by `/path/to/edx-platform`.

Reply:
Sure 😅